pipelines/introduction-to-generic-pipelines/README.md

```diff
@@ -139,7 +141,7 @@ To add a notebook or script to the pipeline:
 
    ![Scan file for environment variables](doc/images/scan-file.png)
 
-   It appears that `load_data` requires environment variable `DATASET_URL` to be set. This variable identifies the name and location of a data set file, which the notebook or script will download and decompress. 
+   It appears that `load_data` references two environment variables i.e `DATASET_URL` and `ELYRA_RUNTIME_ENV`. The `DATASET_URL` requires to be set. This variable identifies the name and location of a data set file, which the notebook or script will download and decompress. The `ELYRA_RUNTIME_ENV` is optional. It can be set to `local`.  
```

Comment:
> The `ELYRA_RUNTIME_ENV` is optional. It can be set to `local`.  

TL;DR. This is a read-only variable. For details refer to https://elyra.readthedocs.io/en/stable/user_guide/best-practices-file-based-nodes.html#proprietary-environment-variables.